libs/execution/src/lib/constraints/executors/length-constraint-executor.ts

```diff
@@ -24,15 +26,25 @@ export class LengthConstraintExecutor implements ConstraintExecutor {
       return false;
     }
 
-    const minLength = context.getPropertyValue(
+    const minLength = context.getPropertyValueOrUndefined(
       'minLength',
       PrimitiveValuetypes.Integer,
     );
-    const maxLength = context.getPropertyValue(
+    const maxLength = context.getPropertyValueOrUndefined(
       'maxLength',
       PrimitiveValuetypes.Integer,
     );
 
-    return minLength <= value.length && value.length <= maxLength;
+    if (minLength !== undefined && maxLength !== undefined) {
+      return minLength <= value.length && value.length <= maxLength;
+    } else if (minLength !== undefined) {
+      return minLength <= value.length;
+    } else if (maxLength !== undefined) {
+      return value.length <= maxLength;
+    }
+
+    // FIXME: throw an actual error?
+    assert(true); // here: minLength === undefined && maxLength === undefined
+    return false; // HACK: linter complains about a missing return type otherwise
```

Comment:
@rhazn I was unsure what to do if no bound is specified

Reply:
As discussed during today's meeting I will handle this by letting all values pass through a constraint without bounds. To compensate this I'm going to add a language server warning.